src/d3a/models/config.py

```diff
@@ -29,7 +29,8 @@
 
 class SimulationConfig:
     def __init__(self, sim_duration: duration, slot_length: duration, tick_length: duration,
-                 market_count: int, cloud_coverage: int, iaa_fee: int,
+                 market_count: int, cloud_coverage: int,
+                 iaa_fee: int=ConstSettings.IAASettings.FEE_PERCENTAGE,
```

Comment:
Sorry, as i am working on the same part of the code I am wondering, whether we want to keep the `iaa_fee` part of the SimulationConfig class. I am asking this because now as he iaa_fee is can also be set individually for each area, and is not a cli parameter any more, we could delete it here. 
The same applies IMO to the `cloud_coverage` and `pv_user_profile` . Is there a plan to move this to the PV class?

Reply:
Hi @hannesdiedrich we should not move these 2 parameters from the config IMO, since these are useful for assigning global weather conditions for the whole grid. Otherwise one would need to upload independent profiles for each PV, instead of just uploading one profile.

Reply:
Ok, thanks! Considering small grids, it makes sense for now.

Reply:
Regarding the iaa_fee though, to expand this discussion a bit more, I wonder if it makes sense to keep it as part of this class, since the user can always access it via ConstSettings.IAASettings.FEE_PERCENTAGE instead of using config.iaa_fee. Config parameters should be IMO only for configuration via d3a-web, otherwise the ConstSettings should be enough. This is a matter of preference though and I do not have a strong opinion on that.

Reply:
Hmm, tricky. It is indeed an important parameter for the simulation, it influences the whole behaviour of the trading. I think the user wants to set it on nearly every different setup. That is why I tend to be in favour of keeping it in the SimulationConfig class. 
It is true, that the user will provide this value for each area though.

Reply:
To summarise the offline discussion with Sarah, there are plans to add this to the UI in the future, therefore I would vote to keep the iaa_fee parameter in the SimulationConfig class.